### PR TITLE
[DOCS] Update detection rule actions import/export docs

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses #1169.

Previews:

* [Manage detection rules | Export and import rules](https://security-docs_1264.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-management.html#import-export-rules-ui) - Updated procedure for export/import.
* [Detections API | Export rules](https://security-docs_1264.docs-preview.app.elstc.co/guide/en/security/master/rules-api-export.html) - Added admonition to beginning of API doc.
* [Detections API | Import rules](https://security-docs_1264.docs-preview.app.elstc.co/guide/en/security/master/rules-api-import.html) - Added admonition to beginning of API doc.

### CODE REVIEW

File: docs/detections/rules-ui-manage.asciidoc [hunk 1]
```diff
@@ -42,8 +42,6 @@ You can then modify the duplicated rules and, if required, delete the prebuilt o
 [[download-prebuilt-rules]]
 === Download latest prebuilt Elastic rules
 
-beta::[]
```
Comment (joepeeples) [inline docs/detections/rules-ui-manage.asciidoc:45]: FYI: This change isn't directly related to rule action export/import, but I noticed the label seemed outdated and confirmed with Garrett Spong that this feature is no longer considered beta.

File: docs/detections/rules-ui-manage.asciidoc [hunk 2]
```diff
@@ -79,20 +77,32 @@ NOTE: For prebuilt rules, you can only activate, deactivate, delete, edit
...
+.. Select *Bulk actions* -> *Export selected*, then save the exported file.
++
+NOTE: You cannot export Elastic prebuilt rules.
+. To import rules:
+.. Click *Import rules*.
+.. Drag and drop the file that contains the detection rules.
 +
-NOTE: You cannot export prebuilt rules.
+NOTE: Imported rules must be in an `.ndjson` file.
+.. Select *Overwrite existing detection rules with conflicting Rule ID* to update existing rules if they match any rules in the import file.
```
Comment (benironside) [inline docs/detections/rules-ui-manage.asciidoc:104]: This didn't make it obvious to me that existing rules' actions will get updated to match the imported rules' actions if both rules have the same criteria. Maybe "...conflicting Rule ID* if you want imported rules to overwrite the actions associated with existing rules when both rules have the same criteria."

**Review by benironside — APPROVED** [review]
LGTM, added one comment at the bottom

File: docs/detections/rules-ui-manage.asciidoc (hunk 2 above)
Comment (joepeeples) [reply docs/detections/rules-ui-manage.asciidoc]: I added another sentence to make this clearer -- check out the latest draft and let me know if that works better. Thanks!

File: docs/detections/rules-ui-manage.asciidoc [hunk 3]
```diff
@@ -79,20 +77,32 @@ NOTE: For prebuilt rules, you can only activate, deactivate, delete, edit
...
-=== Import and export rules
+=== Export and import rules
 
-. Go to *Detect* -> *Rules*.
-. To import rules:
-.. Click *Import rules*.
-.. Drag and drop files that contain the detection rules.
-+
-NOTE: Imported rules must be in an `ndjson` file.
+You can export detection rules to an `.ndjson` file, which you can then import into another {es-sec} environment.
```
Comment (jmikell821) [inline docs/detections/rules-ui-manage.asciidoc:82]:
```suggestion
You can export detection rules to an `.ndjson` file, which you can then import into another {elastic-sec} environment.
```

File: docs/detections/api/rules/rules-api-export.asciidoc [hunk 4]
```diff
@@ -1,12 +1,19 @@
 [[rules-api-export]]
 === Export rules
 
-Exports rules to an ndjson file.
+Exports rules to an `.ndjson` file.
 
 NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
```
Comment (jmikell821) [inline docs/detections/api/rules/rules-api-export.asciidoc:6]:
```suggestion
NOTE: Console supports {es} APIs only. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
```

File: docs/detections/api/rules/rules-api-export.asciidoc [hunk 5]
```diff
@@ -1,12 +1,19 @@
 [[rules-api-export]]
 === Export rules
 
-Exports rules to an ndjson file.
+Exports rules to an `.ndjson` file.
```
Comment (jmikell821) [inline docs/detections/api/rules/rules-api-export.asciidoc:4]:
```suggestion
Exports rules to an `.ndjson` file. Please note that you cannot export prebuilt rules, but they are available at https://github.com/elastic/detection-rules/tree/main/rules/.
```

File: docs/detections/api/rules/rules-api-export.asciidoc [hunk 6]
```diff
@@ -1,12 +1,19 @@
...
 
 NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
 
 NOTE: You cannot export prebuilt rules but they are available at https://github.com/elastic/detection-rules/tree/main/rules/.
```
Comment (jmikell821) [inline docs/detections/api/rules/rules-api-export.asciidoc:8]:
```suggestion

```

File: docs/detections/api/rules/rules-api-import.asciidoc [hunk 7]
```diff
@@ -1,20 +1,27 @@
 [[rules-api-import]]
 === Import rules
 
-WARNING: This API supports {kibana-ref}/api.html#token-api-authentication[Token-based authentication] only.
+Imports rules from an `.ndjson` file.
 
-Imports rules from an ndjson file.
+WARNING: This API supports {kibana-ref}/api.html#token-api-authentication[Token-based authentication] only.
 
 NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
```
Comment (jmikell821) [inline docs/detections/api/rules/rules-api-import.asciidoc:8]:
```suggestion
NOTE: Console supports {es} APIs only. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests]. Please also note this API supports {kibana-ref}/api.html#token-api-authentication[Token-based authentication] only.
```

File: docs/detections/api/rules/rules-api-import.asciidoc [hunk 8]
```diff
@@ -1,20 +1,27 @@
 [[rules-api-import]]
 === Import rules
 
-WARNING: This API supports {kibana-ref}/api.html#token-api-authentication[Token-based authentication] only.
+Imports rules from an `.ndjson` file.
 
-Imports rules from an ndjson file.
+WARNING: This API supports {kibana-ref}/api.html#token-api-authentication[Token-based authentication] only.
```
Comment (jmikell821) [inline docs/detections/api/rules/rules-api-import.asciidoc:6]:
```suggestion

```

**Review by jmikell821 — COMMENTED** [review]
Looks good. Just a few small comments.

**Review by nastasha-solomon — APPROVED** [review]

**Pull request merged**